### ISSUE TITLE
Dependency check ignores Pillow-SIMD

### ISSUE THREAD

**Issue by rharish101** [jbarlow83/OCRmyPDF#546]
**Describe the bug**
I have installed [Pillow-SIMD](https://github.com/uploadcare/pillow-simd), which is the same as Pillow, but it is an optimized version. When I run OCRmyPDF from the CLI, it fails with the following message:
```python
Traceback (most recent call last):
  File "/usr/bin/ocrmypdf", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3259, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3242, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3271, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 584, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 787, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'Pillow>=6.2.0' distribution was not found and is required by ocrmypdf
```

I have installed OCRmyPDF through the AUR for Arch Linux.

**To Reproduce**
Install Pillow-SIMD over Pillow. Then run:
```bash
ocrmypdf  --help
```

**Expected behavior**
The help message should be printed.

**System**
 - OS: Arch Linux
 - OCRmyPDF Version: 9.7.2
 - How did you install ocrmypdf? Did you use a system package manager, `pip`, or a Docker image?: [The Arch User Repository (AUR)](https://aur.archlinux.org/packages/ocrmypdf/)

**Comment by jbarlow83** [comment]
Could you try installing ocrmypdf and Pillow-SIMD into a virtual environment, as described in the installation documentation? 
https://ocrmypdf.readthedocs.io/en/latest/installation.html#installing-with-python-pip

I believe that one would need to modify the ArchLinux PKGBUILD file to do what you are trying to do, to install Pillow-SIMD as a system package. Talk to the ArchLinux packager/maintainer for ocrmypdf about getting Pillow-SIMD to work as an alternate.

As an aside, I don't think Pillow-SIMD will make a measurable difference. ocrmypdf does not use Pillow for very much.

**Comment by rharish101** [comment]
Thanks, the virtualenv thing worked. This is what I did inside a new virtualenv:
```sh
pip install ocrmypdf
pip uninstall Pillow
pip install Pillow-SIMD
```

Everything worked, so I'll contact the AUR maintainer regarding this. Thanks once again!

Regarding the use of Pillow-SIMD, I use Pillow in various other programs, which is why I had installed the SIMD version globally. I thought that the AUR package would pick up on it, but it only did so during installation, not during runtime.
